incubator/vault/values.yaml

```diff
@@ -6,6 +6,21 @@ image:
   repository: vault
   tag: 0.9.0
   pullPolicy: IfNotPresent
+consulAgent:
+  repository: consul
+  tag: 1.0.6
```

Comment:
1.0.7 is now out. Can that newer version be used instead?

Reply:
Yep - I've tested 1.0.7 and updated the default tag